README.md

```diff
@@ -1,6 +1,7 @@
 <img src="/images/reactserver_logo%402x.png" width="450px"/>
 
-# react-server monorepo [![Build Status][build-badge-img]][build-url]
+[![Build Status][build-badge-img]][build-url]
+[![NPM version][npm-badge-img]][npm-url]
```

Comment:
It's the main `react-server` package.  I [argued against it](https://github.com/redfin/react-server/pull/36#discussion_r54039361) at one point, too.

I'll remove it if you think it's misleading.  I just like the look of more badges. :)